WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -66,6 +88,46 @@ class ReaderDetailWebviewViewController: UIViewController, ReaderDetailView {
         }
     }
 
+    /// Adds the sahre button at the right of the nav bar
+    ///
+    private func configureShareButton() {
+        let image = UIImage.gridicon(.shareiOS).withRenderingMode(UIImage.RenderingMode.alwaysTemplate)
+        let button = CustomHighlightButton(frame: CGRect(x: 0, y: 0, width: image.size.width, height: image.size.height))
+        button.setImage(image, for: UIControl.State())
+        button.addTarget(self, action: #selector(didTapShareButton(_:)), for: .touchUpInside)
+
+        let shareButton = UIBarButtonItem(customView: button)
+        shareButton.accessibilityLabel = NSLocalizedString("Share", comment: "Spoken accessibility label")
+        WPStyleGuide.setRightBarButtonItemWithCorrectSpacing(shareButton, for: navigationItem)
+    }
+
+    private func configureHeader() {
+        header.delegate = coordinator
+        headerContainerView.addSubview(header)
+        headerContainerView.pinSubviewToAllEdges(header)
+        headerContainerView.heightAnchor.constraint(equalTo: header.heightAnchor).isActive = true
+        headerContainerView.translatesAutoresizingMaskIntoConstraints = false
+    }
+
+    private func configureDiscoverAttribution(_ post: ReaderPost) {
+        if post.sourceAttributionStyle() == SourceAttributionStyle.none {
```

Comment:
This could be just `if post.sourceAttributionStyle() == .none`

Reply:
Done in 9c07652119c5c27041fe21b2947cc4efa95017b0